lib/assert.js

```diff
@@ -130,6 +132,60 @@ function getBuffer(fd, assertLine) {
   return buffers;
 }
 
+function getErrMessage(call) {
+  const filename = call.getFileName();
+
+  // Skip Node.js modules!
+  if (nativeModules[filename]) {
```

Comment:
You can check the existence of an native module via something like `require('native_module').exists('assert')`, also `require('native_module')._source[id]` actually returns the source if you want to use it..

Reply:
I explicitly want to exclude native source code. I think it could cause more confusion than helping anyone in case users use it like e.g.

```js
const assert = require('assert');
const EventEmitter = require('events');

const e = new EventEmitter();

// Listen and verify the input
e.once('hello', assert);

// Emit some data
e.emit('hello', null);
```